bazel/variables.bzl

```diff
@@ -1,4 +1,37 @@
-DEFAULT_COPTS = [
+# Those are the build flags used to build MalDocA (maldoca/*).
+# The flags are not applied to build third party dependencies.
```

Comment:
Can you add something like: "For chromium-specific flags, see mini_chromium.BUILD."

Reply:
done